src/client.rs

```diff
@@ -500,7 +500,7 @@ async fn test_obfuscated_share_link() {
 #[tokio::test(flavor = "multi_thread")]
 async fn test_share_link_strip_json() {
 	let link = "/17krzvz".into();
-	let canonical_link = "/comments/17krzvz".into();
+	let canonical_link = "/17krzvz".into();
```

Comment:
I left this broken on purpose because it doesn't match the proper behavior btw, visiting redlib instance at /17krzvz indeed fails. So for truth's sake I'd like to leave this one out since it really is "broken".

```suggestion
	let canonical_link = "/comments/17krzvz".into();
```